src/main/java/org/broadinstitute/hellbender/tools/sv/SVDepthOnlyCallDefragmenter.java

```diff
@@ -0,0 +1,120 @@
+     * @param a
+     * @param b
+     * @return true if the two calls should be in the same cluster
+     */
+    @Override
+    protected boolean clusterTogether(final SVCallRecordWithEvidence a, final SVCallRecordWithEvidence b) {
+        if (!isDepthOnlyCall(a) || !isDepthOnlyCall(b)) return false;
+        Utils.validate(a.getContig().equals(a.getEndContig()), "Call A is depth-only but interchromosomal");
+        Utils.validate(b.getContig().equals(b.getEndContig()), "Call B is depth-only but interchromosomal");
+        if (!a.getType().equals(b.getType())) return false;
+        final Set<String> sharedSamples = new LinkedHashSet<>(a.getSamples());
+        sharedSamples.retainAll(b.getSamples());
+        final double sampleOverlap = Math.min(sharedSamples.size() / (double) a.getSamples().size(), sharedSamples.size() / (double) b.getSamples().size());
+        if (sampleOverlap < minSampleOverlap) return false;
+        return getClusteringInterval(a, null)
+                .overlaps(getClusteringInterval(b, null));
+    }
+
+
+    /**
+     * Determine an overlap interval for clustering using {@value #PADDING_FRACTION} padding
+     * Returned interval represents the interval in which the start position of a new event must fall in order to be added to the cluster (including {@param call})
+     * @param call  new event to be clustered
+     * @param currentClusterInterval    the cluster of interest, may be null
+     * @return  an interval describing the cluster after {@param call} is added
+     */
+    @Override
+    protected SimpleInterval getClusteringInterval(final SVCallRecordWithEvidence call, final SimpleInterval currentClusterInterval) {
+        Utils.nonNull(call);
+        final SimpleInterval callInterval = getCallInterval(call);
+        final int paddedCallStart = (int) (callInterval.getStart() - PADDING_FRACTION * callInterval.getLengthOnReference());
+        final int paddedCallEnd = (int) (callInterval.getEnd() + PADDING_FRACTION * callInterval.getLengthOnReference());
+        final String currentContig = getCurrentContig();
+        final int contigLength = dictionary.getSequence(currentContig).getSequenceLength();
+        if (currentClusterInterval == null) {
+            return IntervalUtils.trimIntervalToContig(currentContig, paddedCallStart, paddedCallEnd, contigLength);
+        }
+        //NOTE: this is an approximation -- padding should be based on the length of the call plus currentClusterIntervals
+        final int newMinStart = Math.min(paddedCallStart, currentClusterInterval.getStart());
+        final int newMaxEnd = Math.max(paddedCallEnd, currentClusterInterval.getEnd());
+        return IntervalUtils.trimIntervalToContig(currentContig, newMinStart, newMaxEnd, contigLength);
+    }
+
+    // Not used for single-linkage clustering
+    @Override
+    protected boolean itemsAreIdentical(final SVCallRecordWithEvidence a, final SVCallRecordWithEvidence b) {
+        return false;
```

Comment:
Would it be better to throw an IllegalStateException if we shouldn't reach here?